modules/odc-using-web-terminal.adoc

```diff
@@ -18,4 +18,17 @@ After the Web Terminal Operator is installed, you can use the web terminal as fo
 +
 . Click *Start* to initialize the web terminal using the selected project.
 
++
 After the web terminal is initialized, you can use the preinstalled CLI tools like `oc`, `kubectl`, `odo`, `kn`, `tkn`, `helm`, `kubens`, `subctl`, and `kubectx` in the web terminal.
++
+
+. Click *+* to open multiple tabs within web terminal in the console.
+
++
```

Comment:
```suggestion
```
If you want the note to be left-aligned, you can delete this plus. Otherwise, if you want the note aligned with the step on line 25, keep the plus character and delete line 26.